pkgs/top-level/all-packages.nix

```diff
@@ -14056,6 +14056,8 @@ in
     fftw = fftwSinglePrec;
   };
 
+  rclone = (callPackage ../applications/networking/sync/rclone { }).bin // { outputs = [ "bin" ]; };
```

Comment:
Go packages references in all-packages.nix was changed to default outputs after discussion https://github.com/NixOS/nixpkgs/pull/16129#discussion_r66704198